package NProximity;

message TProximity {
    optional uint32 Shift = 1;
    optional uint32 ShiftLow = 2;
    optional int32 LevelMask = 3;
    optional int32 LevelMaskLow = 4;
    optional int32 LevelIncr = 5;
    optional int32 ShiftedBeg = 6;
    optional int32 ShiftedEnd = 7;
    optional int32 DistanceType = 8;
    optional int32 Level = 9;
    optional int32 Beg = 10;
    optional int32 End = 11;
    optional int32 Center = 12;
}
